sphinx/source/conf.py

```diff
@@ -23,18 +23,18 @@
 # -- Project information -----------------------------------------------------
 
 project = config['sphinx']['name']
+package_dir = config["sphinx"]["package_dir"]
 author = config['metadata']['author']
 copyright = str(date.today().year) + ', ' + author
-package_dir = config["sphinx"]["package_dir"]
 
-sys.path.insert(0, os.path.join(BASE_FOLDER, package_dir))
+sys.path.insert(0, os.path.join(BASE_FOLDER))
```

Comment:
The previous setting of this line was the cause of all the errors while running `sphinx-apidoc -f -o sphinx/source/api src` since this command searches for `src` *under* a path contained in sys.path (not as one of its elements).